Tools/scripts/summarize_stats.py

```diff
@@ -735,8 +758,32 @@ def execution_count_section() -> Section:
         """,
     )
 
+def opcode_input_overlap(
+    uop_flags: dict[str, list[str]], opcode_i: str, opcode_j: str
+) -> str:
+    def flag_compatible(*flags: str):
+        return not (
+            any(f in uop_flags[opcode_i] for f in flags)
+            and any(f in uop_flags[opcode_j] for f in flags)
+        )
+
+    results = (
+        flag_compatible(
+            "HAS_OPARG_AND_1_FLAG",
+        ),
+        flag_compatible(
+            "HAS_OPERAND_FLAG",
+        ),
+        flag_compatible("HAS_JUMP_FLAG", "HAS_EXIT_FLAG", "HAS_DEOPT_FLAG"),
+    )
+    result_names = ("Oparg", "Operand", "Target")
+
+    if results.count(False) == 0:
+        return "No Conflict"
+    return f"Conflict. Both Use: {','.join(result_names[r] for r in range(3) if not results[r])}"
```

Comment:
I think putting the two tuples in a dictionary makes working on them together a little easier.  (A tuple of tuples would probably also be fine here).

```suggestion
    results = {
        "Oparg": flag_compatible(
            "HAS_OPARG_AND_1_FLAG",
        ),
        "Operand": flag_compatible(
            "HAS_OPERAND_FLAG",
        ),
        "Target": flag_compatible("HAS_JUMP_FLAG", "HAS_EXIT_FLAG", "HAS_DEOPT_FLAG"),
    )
    
    if list(results.values()).count(False) == 0:
        return "No Conflict"
    return f"Conflict. Both Use: {','.join(k for k, v in results.items() if not v)}"
```

Reply:
Thank you! Agreed, this is definitely clearere.